solidity/test/staker-rewards/rewardsData.js

```diff
@@ -0,0 +1,93 @@
+const testValues = {
+  merkleObject0: {
```

Comment:
Thoughts on renaming `merkleObject0` and `merkleObject1` to `interval0` and `interval1`? I feel it would make understanding what we are actually allocating in tests a little easier.
```
await allocateTokens(
  testValues.merkleObject0.merkleRoot,
  testValues.merkleObject0.amountToAllocate
)

vs

await allocateTokens(
  testValues.interval0.merkleRoot,
  testValues.interval0.amountToAllocate
)
```

Reply:
Done.